### DISCUSSION TITLE
Pie chart configuring round corners for each one corner

### DISCUSSION THREAD

**Discussion by ssimnv** [recharts/recharts#3814] [Ideas]
Hello Rechart team, 
Hope you are doing well.

We love of your library, but we have one request:
Could we have configuration to create pie chart like that?
![image](https://github.com/recharts/recharts/assets/107298121/36c5e569-c216-4043-b66f-90ea1b44a130)

Thanks!

**Comment by manikamg** [comment]
i also want to create like this kind of pie chart, so please help us for this
